app/src/test/java/org/oppia/android/app/testing/options/OptionsFragmentTest.kt

```diff
@@ -90,6 +103,7 @@ class OptionsFragmentTest {
 
   @Before
   fun setUp() {
+    TestModule.forceEnableLanguageSelectionUi = true
```

Comment:
should we shift this initialization to each test as some require true some false?